### DISCUSSION TITLE
getting Forbidden secure cookie hash key is not set

### DISCUSSION THREAD

**Discussion by Durgam-Lakshmi** [artifacthub/hub#3849] [Q&A]
Iam trying to register a new user using api. I have deployed code in my local system
Forbidden - securecookie: hash key is not set
This is my configuration file
log:
  level: debug
  pretty: true
db:
  host: localhost
  port: "5432"
  database: hub
  user: postgres
server:
  addr: localhost:8000
  metricsAddr: localhost:8001
  shutdownTimeout: 10s
  webBuildPath: ../../web/build
  widgetBuildPath: ../../widget/build
  basicAuth:
    enabled: false
    username: hub
    password: changeme
  cookie:
    hashKey: default-unsafe-key
    secure: false
theme:
  colors:
    primary: "#417598"
    secondary: "#2D4857"
  images:
    appleTouchIcon192: "/static/media/logo192_v2.png"
    appleTouchIcon512: "/static/media/logo512_v2.png"
    openGraphImage: "/static/media/artifactHub_v2.png"
    shortcutIcon: "/static/media/logo_v2.png"
    websiteLogo: "/static/media/logo/artifacthub-brand-white.svg"
  siteName: "Artifact Hub"
  sampleQueries:
    - name: Packages from verified publishers
      querystring: "verified_publisher=true"
    - name: Operators with auto pilot capabilities
      querystring: "capabilities=auto+pilot"
    - name: Helm Charts in the storage category
      querystring: "kind=0&ts_query=storage"

**Comment by tegioz** [comment]
Hi @Durgam-Lakshmi 👋

Have you deployed Artifact Hub locally using the provided Helm chart? It not, please make sure that `allowUserSignUp` is set to `true` in your configuration.

https://artifacthub.io/packages/helm/artifact-hub/artifact-hub?modal=values&path=hub.server.allowUserSignUp

**Reply by Durgam-Lakshmi** [reply]
No I haven't used the helm chart but deployed the source code on local. Thanks @tegioz now my issue got solved.

**Reply by tegioz** [reply]
Awesome, happy to help!

I asked because the chart sets that value to true by default, but if you run it locally from the source code you need to set it on the config file yourself 🙂
